wait-for-deployment

```diff
@@ -36,11 +41,22 @@ get_deployment_jsonpath() {
   kubectl get deployment "${deployment}" -o "jsonpath=${_jsonpath}"
 }
 
-if [[ $# != 1 ]]; then
-  echo "usage: $(basename $0) <deployment>" >&2
+if [[ $# -lt 1 ]]; then
```

Comment:
I prefer letting scripts also fail when surplus arguments are given. In our case, let the script fail if the number of parameters is less than 1 or greater than 2.